community-plugins.json

```diff
@@ -3456,5 +3456,12 @@
         "author": "Aviral Batra",
         "description": "Searches notes based on current line",
         "repo": "aviral-batra/obsidian-power-search"
+    },
+    {
+        "id": "obsidian-kindle-plugin",
+        "name": "Autosuggest",
```

Comment:
At least for me, the word "autosuggest" usually refers to the typeahead component that you see here:

<img width="189" alt="image" src="https://user-images.githubusercontent.com/693981/153413018-4ae74d4f-0313-4041-b7f8-81063ad49535.png">

A more descriptive name might help avoid confusion and help with discovery. Something like `Highlight Connections` or `Highlight unlinked mentions` (that one's pretty boring sounding, I know)

Reply:
At least in the React.js world, [autosuggest](https://react-autosuggest.js.org/) is pretty common, plus the Obsidian API calls them "Suggests."

Regardless, I like the new name and the plugin looks awesome!